draft-ietf-quic-transport.md

```diff
@@ -786,15 +775,30 @@ reaches 2^62 - 1, the sender MUST close the connection without sending a
 CONNECTION_CLOSE frame or any further packets; a server MAY send a Stateless
 Reset ({{stateless-reset}}) in response to further packets that it receives.
 
-For the packet header, the number of bits required to represent the packet
-number are reduced by including only the least significant bits of the packet
-number.
-
-The encoded packet number is protected as described in {{QUIC-TLS}}. Protection
-of the packet number is removed prior to recovering the full packet number. The
-full packet number is reconstructed at the receiver based on the largest packet
-number received on a successfully authenticated packet. Recovering the full
-packet number is necessary to successfully remove packet protection.
+In the QUIC long and short packet headers, the number of bits required to
+represent the packet number are reduced by including only the least
+significant bits of the packet number.
+
+In the long packet header, the least significant 32 bits are used. In the
```

Comment:
"In the long packet header, the least significant 32 bits of the packet are encoded into the Packet Number field."